src/Raven.Server/Documents/Handlers/TimeSeriesHandler.cs

```diff
@@ -644,6 +644,40 @@ public async Task RemoveTimeSeriesPolicy()
         }
 
         [RavenAction("/databases/*/timeseries/names/config", "POST", AuthorizationStatus.ValidUser)]
+
+        [RavenAction("/databases/*/timeseries/debug", "GET", AuthorizationStatus.ValidUser)]
```

Comment:
there might be more debug endpoints.
let's call this one:
`/databases/*/timeseries/debug/segments-summery`